### PR TITLE
#118: fixed a few sonarqube code smell issues

### PR DESCRIPTION
**Pull request by tarekait1996**
removed about 30 code smells just to decrease the number a bit

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -1,2 +1,2 @@
-include ':app', ':google-maps-services-java-master', ':google-maps-services-java-0.10.0'
+include ':app', ':google-maps-services-java-0.10.0'
```
Comment (Hambrsoom) [inline settings.gradle]: did you test it? it works fine without any issues?

File: build.gradle [hunk 2]
```diff
@@ -25,7 +25,8 @@ sonarqube {
         property "sonar.test.inclusions", "**/*Test*/**"
         property "sonar.sourceEncoding", "UTF-8"
         property "sonar.sources", "src/main/java"
-        property 'sonar.coverage.exclusions', ["**/Activities/**", "**/Adapters/**", "**/Database/Daos/**", "**/Global/**", "**/*Fragment.java", "**/ClassConstants.*", '**/TransitType.*']
+        property 'sonar.exclusions', ['**/GoogleMapsServicesTools/**', "**/*GoogleMapsServicesTools*/**", "**/*google-maps-services-java-0.10.0*/**"]
+        property 'sonar.coverage.exclusions', ['**/GoogleMapsServicesTools/**', "**/*GoogleMapsServicesTools*/**", "**/*google-maps-services-java-0.10.0*/**", "**/Activities/**", "**/Adapters/**", "**/Database/Daos/**", "**/Global/**", "**/*Fragment.java", "**/ClassConstants.*", '**/TransitType.*']
```
Comment (ferdousara-parvin) [inline build.gradle]: ok so you were able to exclude the code smells coming from the library?

**Comment by ferdousara-parvin** [comment]
Ran the app, everything seems to work fine!

File: settings.gradle (hunk 1 above)
Comment (tarekait1996) [reply settings.gradle]: Ferdousara and I tried it and it seems to not affect the app

File: build.gradle (hunk 2 above)
Comment (tarekait1996) [reply build.gradle]: exact!

**Pull request merged**